lib/middleman-images/image.rb

```diff
@@ -25,7 +63,9 @@ def process
         app.logger.info "== Images: Processing #{@path}"
 
         FileUtils.copy(original_source_file, processed_source_file)
-        resize(processed_source_file, @processing_options[:resize]) unless @processing_options[:resize].nil?
+        run_mini_magick = MINI_MAGICK_OPTIONS.find { |o| @processing_options[o] }
```

Comment:
Would this work as well:
```suggestion
        run_mini_magick = MINI_MAGICK_OPTIONS.find { |o| @processing_options.has_key?(o) }
```
?
Or is there an option that could be actually the ruby value `false` and should not trigger MiniMagick running?

Reply:
They are doing different things if an argument is `nil`. But maybe this is even wanted? :thinking: